README.md

```diff
@@ -445,7 +445,7 @@ This feature allows you to receive events which correspond to your ad revenue th
 ```objectivec
 [TenjinSDK init:@"TENJIN_API_KEY"];
 [TenjinSDK subscribeAppLovinImpressions];
-
+[TenjinSDK appLovinImpressionFromJSON:(NSString)jsonString];
```

Comment:
JSON public methods are not required, if we are able to use the other methods in the internal test app for getting ILRD data.
@IrminDzevlan Please re-check the internal test app and update the PR